static/templates/settings/organization_profile_admin.hbs

```diff
@@ -22,6 +22,13 @@
                     <textarea id="id_realm_description" name="realm_description" class="admin-realm-description setting-widget prop-element"
                       maxlength="1000" data-setting-widget-type="string">{{ realm_description }}</textarea>
                 </div>
+                <div class="input-group">
+                    <label for="realm_oss_showcase_policy">{{t "Whether the organisation is publicly advertised on zulip.com/open-communities" }}</label>
+                    <select name="realm_oss_showcase_policy" class="setting-widget prop-element" id="id_realm_oss_showcase_policy" data-setting-widget-type="integer">
```

Comment:
This should be `data-setting-widget-type="number"`.

Reply:
And cant we use directly value="1" and value="2" for options. Is there any reason you are using strings. We use the integer values for other policies like stream creation, etc.